README.md

```diff
@@ -228,7 +246,7 @@ endfunction
 function! s:FZFSelectKit() abort
```

Comment:
I propose to remove this example because you already have implemented: `fzf#cmake4vim#SelectKit()`

Moreover could you double check that we don't have redundant examples?

Reply:
This is only for the case you want to add more entries into `g:cmake_kits` which are not supported by the plugin.

E.g. if you're using `YouCompleteMe` and developing with `emscripten`, you could pass `em++` to `clangd` via `g:ycm_clangd_args`. But I didn't want to add this functionality to the plugin, because you don't have to use both `cmake4vim` *and* `YouCompleteMe`, but in case you do (like many people do, myself incuded), I wrote these instructions to help people.

And if someone wants to do something like that, you can't override the plugin directly (in `~/.vim/plugged/cmake4vim`, or wherever your plugin is), because then you can't update your plugin, or the update will revert your changes (depends on the plugin manager), so it has to go to the `~/.vim/after` folder, so it gets applied _after_ the plugin initializes.